src/java.base/share/classes/sun/nio/fs/AbstractPoller.java

```diff
@@ -54,22 +52,14 @@ protected AbstractPoller() {
     /**
      * Starts the poller thread
      */
-    @SuppressWarnings("removal")
     public void start() {
-        final Runnable thisRunnable = this;
-        AccessController.doPrivileged(new PrivilegedAction<>() {
-            @Override
-            public Object run() {
-                Thread thr = new Thread(null,
-                                        thisRunnable,
-                                        "FileSystemWatchService",
-                                        0,
-                                        false);
-                thr.setDaemon(true);
-                thr.start();
-                return null;
-            }
-         });
+        Thread thr = new Thread(null,
+                                this,
+                                "FileSystemWatchService",
+                                0,
+                                false);
+        thr.setDaemon(true);
+        thr.start();
```

Comment:
Not for this PR but we should replace this with an InnocuousThread at some point.

Reply:
Tracked by [JDK-8345432](https://bugs.openjdk.org/browse/JDK-8345432).